index.js

```diff
@@ -340,6 +344,16 @@ const plugin = fp(async function (app, opts) {
     fastifyGraphQl.defineResolvers(resolvers)
   }
 
+  fastifyGraphQl.transformSchema = function (transforms) {
+    if (!Array.isArray(transforms)) {
+      transforms = [transforms]
+    }
+
+    for (const fn of transforms) {
+      fastifyGraphQl.replaceSchema(fn(fastifyGraphQl.schema))
```

Comment:
`fn` seems oddly unspecifc, I believe it would help with the readability of the code if that would be renamed to `transformFn` or something like that.